bin/docker_run.sh

```diff
@@ -12,4 +12,9 @@ if [ $? -ne 0 ]; then
     schematic migrations/
 fi
 
+if [ "$BRAINTREE_MERCHANT_ID" ]; then
+    echo "Braintree merchant ID found in environment, running braintree_config."
+    python manage.py braintree_config
```

Comment:
hmm, it was probably an artifact of before we switched around the public_id. I deleted everything and was able to run it multiple times after that.